src/exo6 - ReaderTaskEither/exo6.exercise.ts

```diff
@@ -23,13 +24,18 @@ import { User } from './domain';
 // current context. In the following example, we need to fetch a user by its id,
 // and then we want to return its name capitalized.
 
-export const getCapitalizedUserName: (args: {
+const capitalizeName = (name: string) => {
+  return `${name[0].toUpperCase() + name.slice(1)}`;
```

Comment:
Usually, to concatenate strings we will rather use this syntax below and avoid using `+`. Also to get a specific character in a string, `char` is recommended rather than considering the string as an array.
So it would look like this:
```ts
`${name.charAt(0).toUpperCase()}${name.slice(1)}`
```